script/Arcueid/Day 3/03_00_ARC03_6.txt

```diff
@@ -107,6 +107,7 @@ Why am I back here, sleeping in my own bed?
 [sha:c337ca4ee7ff6930288fc0590233ff238f1edade]{
 -- Page 278, Offset 4763.
 -- 「俺は―――ひとを、ころし――」
+// ughh
 "I――that woman――I ki―――"
```

Comment:
Honestly, this could even be something like
>I... I just kill―――

Follow up needs a bit changing

>I clamp my mouth shut before I can finish the sentence.